src/util/grammar-patch.ts

```diff
@@ -17,8 +17,9 @@ export function grammarPatch (grammar: Grammar, patch: Grammar = grammar): Gramm
 
 			if (key?.includes('/')) {
 				// Deep key
-				const path = key.split('/');
+				let path = key.split('/');
 				const lastKey = path.pop();
+				path = path.flatMap(key => [key, 'inside']); // add `inside` after each key
```

Comment:
Is this something you had in mind to transform deep keys like `foo/bar/baz/yolo` into

```js
'foo': {
	inside: {
		'bar': {
			inside: {
				'baz': {
					inside: {
						'yolo': {...}
					}
				}
			}
		}
	}
}
```

Reply:
> Possibly? Does it work?

It does.

> Also, can you check if we have any case where we may want to insert something at the same level as the object that has the `inside` key?

Will do